### ISSUE TITLE
[Cinnamon] Bluetooth tray icon does not adapt to the theme

### ISSUE THREAD

**Issue by mantuoliveira** [linuxmint/mint21-beta#93]
**Describe the bug**

When changing from a dark to a light Mint-Y theme, the tray icons adapt their color accordingly. However, this does not happen for the Bluetooth icon. 

**To Reproduce**

1 - Change the theme from Mint-Y-Dark-<color> to Mint-Y-<color> while having the Bluetooth tray icon visible

**Expected behavior**

On a light background, the icon should become dark in order to keep it visible.

**Screenshots**

[Video](https://youtu.be/6GkDu48Ceq4)

**Hardware and mintUpdate**

The system is up to date.

System:
  Kernel: 5.15.0-41-generic x86_64 bits: 64 compiler: gcc v: 11.2.0
    Desktop: Cinnamon 5.4.4 tk: GTK 3.24.33 vt: 7 dm: LightDM 1.30.0
    Distro: Linux Mint 21 Vanessa base: Ubuntu 22.04 jammy
Machine:
  Type: Laptop System: Acer product: Nitro AN515-44 v: V1.04
    serial: <superuser required>
  Mobo: RO model: Stonic_RNS v: V1.04 serial: <superuser required>
    UEFI: Insyde v: 1.04 date: 02/04/2021
Battery:
  ID-1: BAT1 charge: 57.5 Wh (100.0%) condition: 57.5/58.8 Wh (97.9%)
    volts: 17.6 min: 15.4 model: SMP AP18E7M type: Li-ion serial: <filter>
    status: Full
CPU:
  Info: 6-core model: AMD Ryzen 5 4600H with Radeon Graphics bits: 64
    type: MT MCP smt: enabled arch: Zen 2 rev: 1 cache: L1: 384 KiB L2: 3 MiB
    L3: 8 MiB
  Speed (MHz): avg: 1598 high: 2136 min/max: 1400/3000 boost: enabled
    cores: 1: 1431 2: 1506 3: 1430 4: 1802 5: 1421 6: 1562 7: 1570 8: 1540
    9: 1519 10: 2136 11: 1518 12: 1741 bogomips: 71866
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm
Graphics:
  Device-1: NVIDIA TU117M vendor: Acer Incorporated ALI driver: N/A pcie:
    speed: 8 GT/s lanes: 8 bus-ID: 01:00.0 chip-ID: 10de:1f99 class-ID: 0300
  Device-2: AMD Renoir vendor: Acer Incorporated ALI driver: amdgpu
    v: kernel pcie: speed: 16 GT/s lanes: 16 ports: active: eDP-1 empty: none
    bus-ID: 05:00.0 chip-ID: 1002:1636 class-ID: 0300
  Device-3: Chicony HD User Facing type: USB driver: uvcvideo bus-ID: 3-3:2
    chip-ID: 04f2:b64f class-ID: 0e02 serial: <filter>
  Display: x11 server: X.Org v: 1.21.1.3 driver: X: loaded: amdgpu,ati
    unloaded: fbdev,modesetting,vesa gpu: amdgpu display-ID: :0 screens: 1
  Screen-1: 0 s-res: 1920x1080 s-dpi: 96 s-size: 508x285mm (20.0x11.2")
    s-diag: 582mm (22.9")
  Monitor-1: eDP res: 1920x1080 dpi: 142 size: 344x194mm (13.5x7.6")
    diag: 395mm (15.5")
  OpenGL: renderer: AMD RENOIR (LLVM 13.0.1 DRM 3.42 5.15.0-41-generic)
    v: 4.6 Mesa 22.0.1 direct render: Yes
Audio:
  Device-1: NVIDIA vendor: Acer Incorporated ALI driver: snd_hda_intel
    v: kernel pcie: speed: 8 GT/s lanes: 8 bus-ID: 01:00.1 chip-ID: 10de:10fa
    class-ID: 0403
  Device-2: AMD Raven/Raven2/FireFlight/Renoir Audio Processor
    vendor: Acer Incorporated ALI driver: N/A pcie: speed: 16 GT/s lanes: 16
    bus-ID: 05:00.5 chip-ID: 1022:15e2 class-ID: 0480
  Device-3: AMD Family 17h HD Audio vendor: Acer Incorporated ALI
    driver: snd_hda_intel v: kernel pcie: speed: 16 GT/s lanes: 16
    bus-ID: 05:00.6 chip-ID: 1022:15e3 class-ID: 0403
  Sound Server-1: ALSA v: k5.15.0-41-generic running: yes
  Sound Server-2: PulseAudio v: 15.99.1 running: yes
  Sound Server-3: PipeWire v: 0.3.48 running: yes
Network:
  Device-1: Realtek vendor: Acer Incorporated ALI driver: r8169 v: kernel
    pcie: speed: 2.5 GT/s lanes: 1 port: 2000 bus-ID: 03:00.0
    chip-ID: 10ec:2600 class-ID: 0200
  IF: enp3s0 state: down mac: <filter>
  Device-2: Intel Wi-Fi 6 AX200 driver: iwlwifi v: kernel pcie:
    speed: 5 GT/s lanes: 1 bus-ID: 04:00.0 chip-ID: 8086:2723 class-ID: 0280
  IF: wlp4s0 state: up mac: <filter>
Bluetooth:
  Device-1: Intel AX200 Bluetooth type: USB driver: btusb v: 0.8
    bus-ID: 1-4:2 chip-ID: 8087:0029 class-ID: e001
  Report: hciconfig ID: hci0 rfk-id: 2 state: up address: <filter>
    bt-v: 3.0 lmp-v: 5.2 sub-v: 237e hci-v: 5.2 rev: 237e
Drives:
  Local Storage: total: 476.94 GiB used: 21.43 GiB (4.5%)
  ID-1: /dev/nvme0n1 vendor: Western Digital
    model: PC SN530 SDBPNPZ-512G-1014 size: 476.94 GiB speed: 31.6 Gb/s
    lanes: 4 type: SSD serial: <filter> rev: 21103900 temp: 36.9 C
    scheme: GPT
Partition:
  ID-1: / size: 467.89 GiB used: 21.42 GiB (4.6%) fs: ext4
    dev: /dev/nvme0n1p2
  ID-2: /boot/efi size: 511 MiB used: 5.2 MiB (1.0%) fs: vfat
    dev: /dev/nvme0n1p1
Swap:
  ID-1: swap-1 type: file size: 2 GiB used: 0 KiB (0.0%) priority: -2
    file: /swapfile
Sensors:
  System Temperatures: cpu: N/A mobo: N/A gpu: amdgpu temp: 44.0 C
  Fan Speeds (RPM): N/A
Repos:
  Packages: apt: 2269
  No active apt repos in: /etc/apt/sources.list
  Active apt repos in: /etc/apt/sources.list.d/google-chrome.list
    1: deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main
  Active apt repos in: /etc/apt/sources.list.d/official-package-repositories.list
    1: deb http://packages.linuxmint.com vanessa main upstream import backport
    2: deb http://archive.ubuntu.com/ubuntu jammy main restricted universe multiverse
    3: deb http://archive.ubuntu.com/ubuntu jammy-updates main restricted universe multiverse
    4: deb http://archive.ubuntu.com/ubuntu jammy-backports main restricted universe multiverse
    5: deb http://security.ubuntu.com/ubuntu/ jammy-security main restricted universe multiverse
Info:
  Processes: 312 Uptime: 8m wakeups: 1 Memory: 15 GiB used: 2.45 GiB (16.3%)
  Init: systemd v: 249 runlevel: 5 Compilers: gcc: 11.2.0 alt: 11 Shell: Bash
  v: 5.1.16 running-in: gnome-terminal inxi: 3.3.13

**Additional context**

It works fine for other tray icons.
After restarting the machine, the icon presents itself correctly.

**Comment by mtwebster** [comment]
I can't reproduce this with the xapp status applet - switching between light and dark themes inverts the icon colors properly.

Does this always happen? I've noticed once or twice that the bluetooth icon got switched from the xapp status applet to the old systray applet (which doesn't switch colors properly).  Can you try restarting the bluetooth service? You should be able to right-click and exit, then restart it from the menu.

**Comment by mantuoliveira** [comment]
When I found this issue, I was able to reproduce it 100% of the time. However, today, after reading your comment, I've tried again and the icon colors are properly inverted.

I think the issue is what you have pointed out. Rewatching the video I made, it seams that the Bluetooth icon isn't grouped in the xapp status applet (It shows on the left of OBS studio's icon). I'm not able, however, to find what triggers this.
